main/lsp/ShowOperation.cc

```diff
@@ -0,0 +1,29 @@
+#include "main/lsp/ShowOperation.h"
+#include "main/lsp/LSPConfiguration.h"
+#include "main/lsp/LSPMessage.h"
+#include "main/lsp/LSPOutput.h"
+
+namespace sorbet::realmain::lsp {
+using namespace std;
+
+unique_ptr<LSPMessage> makeShowOperation(std::string_view operationName, std::string_view description,
+                                         SorbetOperationStatus status) {
+    return make_unique<LSPMessage>(make_unique<NotificationMessage>(
+        "2.0", LSPMethod::SorbetShowOperation,
+        make_unique<SorbetShowOperationParams>(string(operationName), string(description), status)));
+}
+
+ShowOperation::ShowOperation(LSPOutput &output, const LSPConfiguration &config, std::string_view operationName,
```

Comment:
[a niptick, happy either way]
as you're going to copy `operationName` anyway, it would be better to take `std::string` by copy and move it.